kinto/core/resource/schema.py

```diff
@@ -6,6 +6,10 @@
                                FieldList, TimeStamp, URL)
 from kinto.core.utils import native_value
 
+POSTGRESQL_MAX_INTEGER_VALUE = 2**64 / 2
+
+POSITIVE_BIG_INTEGER = colander.Range(min=0, max=POSTGRESQL_MAX_INTEGER_VALUE)
```

Comment:
In other parts of code we used lowercase for validators, but anything is fine, i guess.